StRoot/StEvent/StFstRawHit.h

```diff
@@ -22,6 +22,8 @@ class StFstRawHit : public StObject
       const Container &charges, const Container &chargeErrs = Container{},
       UChar_t maxTimeBin = 1, UShort_t idTruth = 0);
 
+   StFstRawHit(const StFstRawHit *rawHit);
```

Comment:
This is not a copy constructor. The signature for the copy constructor is: `StFstRawHit(const StFstRawHit& rawHit);`

Reply:
Thanks for correction. Changed it.